src/main/kotlin/caculator/StringAddCalculator.kt

```diff
@@ -0,0 +1,15 @@
+package caculator
+
+internal class StringAddCalculator(private val tokenParser: TokenParser) {
+
+    internal fun add(input: String?): Int {
+        if (input.isNullOrEmpty()) {
+            return EMPTY_VALUE
+        }
+        return tokenParser.parseToken(input).sum()
```

Comment:
> StringAddCalculator을 사용하는 클라이언트가 Number을 모르도록 캡슐화 하기 위해 Int로 return 하였습니다.
개인적으로 Numbers.sum()에서 int타입으로 return 해주는게 좋을거 같습니다. StringAddCalculator가 Number가 아닌 Numbers에만 의존성을 가지도록 제한할수 있다고 생각합니다.
의견 부탁드립니다.

StringAddCalculator가 Numbers에만 의존성을 가지도록 제한하면 어떤 이점이 있을까요??

Reply:
one line으로 code를 작성하다보면 외부 사람들이 봤을 때 해당 코드가 Numbers에 대한 의존을 가지고 있는지 한눈에 보기 힘들것 같아요!! 아래와 같이 두개의 라인으로 나누어 본다면 가독성 측면에서 더 좋지 않을까요?
```suggestion
        val numbers = tokenParser.parseToken(input)
        return numbers.sum()
```